app/uk/gov/hmrc/excisemovementcontrolsystemapi/scheduling/PushNotificationJob.scala

```diff
@@ -39,25 +39,30 @@ class PushNotificationJob @Inject() (
 
   override def name: String = "push-notification-job"
 
-  override def execute(implicit ec: ExecutionContext): Future[Done] =
+  override def execute(implicit ec: ExecutionContext): Future[Done] = {
+    logger.warn(s"[PushNotificationJob] - Started PushNotificationJob")
+
     movementRepository.getPendingMessageNotifications
       .flatMap { notifications =>
         notifications.traverse { notification =>
           processNotification(notification).recover { case NonFatal(_) =>
-            logger.info(
-              s"Failed to notify ${notification.recipient} for message ${notification.messageId}. Will try again later."
+            logger.warn(
+              s"[PushNotificationJob] - Failed to notify ${notification.recipient} for message ${notification.messageId}. Will try again later."
```

Comment:
This is still logging the Id's.  Probably good to have the messageId but the recipient Id will be an ERN I think

Reply:
Done :)